pint_server/pint_db_migrate/env.py

```diff
@@ -0,0 +1,81 @@
+from models import AmazonServersModel
+from logging.config import fileConfig
+
+from sqlalchemy import engine_from_config
+from sqlalchemy import pool
+
+from alembic import context
+
+# this is the Alembic Config object, which provides
+# access to the values within the .ini file in use.
+config = context.config
+
+# Interpret the config file for Python logging.
+# This line sets up loggers basically.
+#fileConfig(config.config_file_name)
+
+
+
+
+# add your model's MetaData object here
+# for 'autogenerate' support
+# from myapp import mymodel
+# target_metadata = mymodel.Base.metadata
+target_metadata = AmazonServersModel.metadata
```

Comment:
We should be use Base.metadata here.

```
from pint_server.models import Base
target_metadata = Base.metadata
```

Reply:
It doesn't matter which model we use, once the PR with the `Base` one gets merged, I will solve the conflict with `Base` as well